website/docs/gitops-run/get-started.mdx

```diff
@@ -0,0 +1,266 @@
+you have full `cluster-admin` privileges.
+
+<Tabs>
+<TabItem value="kind">
+
+Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and run
+
+```bash
+kind create cluster
+```
+</TabItem>
+<TabItem value="k3d">
+
+Install [k3d](https://k3d.io/) and run
+
+```bash
+k3d cluster create mycluster
+```
+</TabItem>
+<TabItem value="minikube">
+
+Install [minikube](https://minikube.sigs.k8s.io/docs/start/) and run
+
+```bash
+minikube start
+```
+</TabItem>
+<TabItem value="docker desktop">
+
+Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and enable Kubernetes. Then run
+
+```
+kubectl config set-context docker-desktop
+```
+</TabItem>
+<TabItem value="other">
+
+GitOps Run works on any kubernetes platform, but to avoid accidents
+you have to explicitly white-list the context name.
+
+First, find the name of the context where you want to run `gitops beta run` - in this example, there's a cluster with the name "dev":
+
+```bash
+$ kubectl config get-contexts
+CURRENT   NAME                      CLUSTER                   AUTHINFO                  NAMESPACE
+*         dev                       dev                       dev
+```
+
+Then, for any `gitops beta run` command in this guide, you'll have to add the flag `--allow-k8s-context=dev`
+</TabItem>
+</Tabs>
+
+Before you continue, make sure `kubectl get nodes` returns a node which is `Ready`.
+
+### Git repository
+
+You need to set up a Git repository to put your GitOps manifests
+in. Any Git repository would do, for example create a new
+[github](https://github.com/new) repository and clone that.
+
+### Set up GitOps Run
+
+To start GitOps Run, point it at the directory you want to keep your
+application manifests. In this example, we will install a small demo application
+[podinfo](https://github.com/stefanprodan/podinfo).
+
+We will start it with `--no-session` as it's a single user
+cluster which we want to use in direct mode. The port-forward points
+at the podinfo pod we will create later.
+
+```
+gitops beta run ./podinfo --no-session --port-forward namespace=dev,resource=svc/dev-podinfo,port=9898:9898
+```
+
+You will now be asked if you want to install Flux and the GitOps
+[dashboard](../getting-started.mdx). Answer yes and set a password, and
+shortly after you should be able to [open the dashboard](http://localhost:9001)
+to see what's in your cluster - including the resources that GitOps
+run is operating.
+
+### Start modifying
+
+GitOps run will have created a directory `podinfo` in your
+current directory. Inside that directory, there will only be a single
+file `kustomization.yaml`.
+
+To create the podinfo automation, we have to create the resources to
+run podinfo - we'll create a new `Namespace`, a `HelmRepository` that
+references the Helm repository where the manifests are stored, and a
+`HelmRelease` that references the chart and version. We can use the
+`flux` CLI to generate the resource definition, or we can just create
```

Comment:
Do we need the comma before "or"? It's a regular list with two items, I suppose.